src/styles/slingshot/main.scss

```diff
@@ -36,8 +36,8 @@
 	display: none !important;
 }
 
-.responsive-img,
-.article-container > img {
+article img { 
+    max-width: 100%;
```

Comment:
Keep in mind this is a different theme for Slingshot docs, I'd be cautious not to break anything there.
Perhaps keep responsive-img class just in case?

```
.responsive-img,
article img { 
        max-width: 100%;
        box-shadow: none !important;
	border-radius: $global-radius;
	margin-bottom: 16px;
}
```

Reply:
I think that everything will keep working correctly.